local-pv-provisioner/controllers/device_detector.go

```diff
@@ -22,21 +25,108 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/metrics"
 )
 
+//+kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch
+
 const (
 	// StorageClass is the name of StorageClass. It is set to pv.spec.storageClassName.
 	StorageClass = "local-storage"
 
 	localPVProvisionerLabelKey = "local-pv-provisioner.cybozu.com/node"
+
+	lppAnnotPrefix           = "local-pv-provisioner.cybozu.io/"
```

Comment:
How about rename `lppAnnotPrefix` to `lppPrefix` or `lppDomain`? The main reason is that this prefix is used both for annotation and label (`lppAnnotNode`).

In addition, I suggest renaming `localPVProvisionerLabelKey` to `localPVProvisonerLegacyLabelKey` or `lppLegacyLabelKey`.